testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/LabAuthenticationHelper.java

```diff
@@ -37,15 +42,32 @@
 
     private String mAppId;
     private String mAppSecret;
+    private final String mKeyVaultSecret;
 
     private static LabAuthenticationHelper sLabAuthHelper;
 
+    private LabAuthenticationHelper(final String keyVaultSecret) {
+        mAppId = null;
+        mAppSecret = null;
+        this.mKeyVaultSecret = keyVaultSecret;
+    }
+
     private LabAuthenticationHelper() {
         mAppId = null;
         mAppSecret = null;
+        String buildConfigSecret = com.microsoft.identity.internal.testutils.BuildConfig.LAB_CLIENT_SECRET;
+        mKeyVaultSecret = buildConfigSecret;
```

Comment:
These two lines can be collapsed into one

Reply:
done